applications/DG-Max/Algorithms/DivDGMaxEigenvalue.cpp

```diff
@@ -36,25 +36,25 @@ ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-#include "DivDGMaxEigenValue.h"
+#include "DivDGMaxEigenvalue.h"
 
-#include "../DGMaxLogger.h"
+#include <valarray>
```

Comment:
why valarray ?

Reply:
Because that has a nicer interface than vector for this usecase. The basic need is the numbers `offset + (0...n-1)`, where the offset changes often but n remains constant. So I used valarray to change the offset.

Looking at it now I don't think I would do it the same way again. But changing it is also not really worth it in my opinion.

Reply:
okay.